spec/features/users/user_deletes_account_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+# As an authenticated user
+# I want to delete my account
+# So that my information is no longer retained by the app
+
+feature "user deletes account", %{
+  As a user, I want to delete my account
+  so that my information is no longer retained by the app
+} do
+  scenario "user deletes account" do
+    visit new_user_registration_path
+    fill_in "Email", with: "john@example.com"
+    fill_in "Password", with: "password"
+    fill_in "Password confirmation", with: "password"
+
+    click_button "Sign up"
```

Comment:
You can use FactoryGirl to create a user object and then navigate to the delete page and click delete. This might be better than signing up to delete an account.